### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by arrudacesar**
A new Issue template for rdstation-ruby-client

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+---
+
+# Prerequisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
```
Comment (filipenasc) [inline .github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md]: We can remove this part: `(for multi-repository projects)`. What do you think?

File: .github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Please describe the behavior you are expecting
+
+# Current Behavior
+
+What is the current behavior?
+
+# Failure Information (for bugs)
+
+Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.
```
Comment (filipenasc) [inline .github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md]: I think we can use either `Please help to provide...` or `Please provide...`

File: .github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md (hunk 1 above)
Comment (arrudacesar) [reply .github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md]: Nice suggestion ;)

File: .github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md (hunk 2 above)
Comment (arrudacesar) [reply .github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md]: Done ;)

**Pull request merged**